public/app/features/search/page/state/actions.ts

```diff
@@ -1,27 +1,12 @@
-import { reduceField } from '@grafana/data';
 import { ThunkResult } from 'app/types';
-import { getRawIndexData, getFrontendGrafanaSearcher } from '../../service/frontend';
 import { fetchResults } from './reducers';
 
 export const loadResults = (query: string): ThunkResult<void> => {
   return async (dispatch) => {
-    const data = await getRawIndexData();
-    if (!data.dashboard) {
-      return;
-    }
-
-    const searcher = getFrontendGrafanaSearcher(data);
-    const results = await searcher.search(query);
-
-    // HACK avoid redux error!
-    results.body.fields.forEach((f) => {
-      reduceField({ field: f, reducers: ['min', 'max'] });
-    });
-
     return dispatch(
       fetchResults({
         data: {
-          results,
+          results: {} as any,
```

Comment:
we should know the result type right?

Reply:
Actually looking more into this it looks like we aren't using redux for search result state at all anymore, so we can probably clean that up as part of these changes as well